locale/zh-cn/about/index.md

```diff
@@ -25,23 +25,20 @@ server.listen(port, hostname, () => {
 });
 ```
 
-这与今天使用 OS 线程的更常见并发模型形成了对比。基于线程的网络效率相对低下，使用起来非常困难。此外，Node.js 的用户不必担心死锁过程，因为没有锁。Node 中几乎没有函数直接执行 I/O 操作，因此进程从不阻塞。由于没有任何阻塞，可伸缩系统在 Node 中开发是非常合理的。
+这与当今比较常见的采用操作系统线程的并发模型形成了鲜明对比。基于线程的网络效率相对较低且更难以使用。此外，由于没有锁，Node.js 的用户不用担心进程死锁的问题。Node.js 中几乎没有函数直接执行 I/O，所以进程从不阻塞，因为没有任何阻塞，所以用 Node.js 来开发可扩展系统是非常合理的。
```

Comment:
`所以进程从不阻塞` 中的 `所以` 可以去掉。否则与后面的因为所以不搭配。

Reply:
Almost no function in Node.js directly performs I/O, so the process never blocks. Because nothing blocks, scalable systems are very reasonable to develop in Node.js.

Node.js 中几乎没有函数直接执行 I/O，进程就不会被阻塞，因为没有任何阻塞，所以用 Node.js 来开发可扩展系统是非常合理的。